docs/topics/Gateway.md

```diff
@@ -38,38 +38,37 @@ Returns an object with the same information as [Get Gateway](#DOCS_GATEWAY/get-g
 The Discord Gateway has a versioning system which is separate from the core APIs. The following table specifies all versions of the Gateway API that have been officially supported, and whether or not they are out of service (e.g. unsupported and potentially disfunctional). The documentation herein is only for the latest version in the following table, unless otherwise specified.
 
 | Version | Out of Service |
-|------------|----------------|
+|---------|----------------|
 | 5 | no |
 | 4 | no |
 
-
 ## Gateway OP Codes/Payloads
 
 ###### Gateway Payload Structure
 
 | Field | Type | Description | Present |
-|-------|------|-------------|-------------|
+|-------|------|-------------|---------|
 | op | integer | opcode for the payload | Always |
-| d | mixed (object, integer) | event data | Always |
+| d | mixed (object, integer, bool, null) | event data | Usually |
```

Comment:
null is not a type, null is a value.

where is `d` not present in the payload? `null` is a valid value of `d`